.github/workflows/integrate-and-deploy.yml

```diff
@@ -46,11 +46,18 @@ jobs:
       -
         name: Establish image name
         id: image
+        env:
+          TAG_LFMERGE:
         run: |
           echo ::set-output name=NAMESPACE::sillsdev/web-languageforge
           echo ::set-output name=TAG_APP::${{ inputs.image-tag-app }}
           echo ::set-output name=TAG_PROXY::${{ inputs.image-tag-proxy }}
           echo ::set-output name=TAG_NEXT_APP::${{ inputs.image-tag-next-app }}
+          echo ::set-output name=LFMERGE_NAMESPACE::ghcr.io/sillsdev/lfmerge
+          # LfMerge tag from parameter if provided, use .json file if exists, fallback to "latest" as default
+          TAG_LFMERGE=$(head -n 1 docker/lfmerge/Dockerfile | cut -d: -f2)
```

Comment:
`head -1` also works that same if you like the shorter syntax

Reply:
I do know that, but I've always preferred the `-n num` syntax. For one thing, if you want to specify a negative number then you must use the `-n` syntax (`head -n -1` displays all but the last line of the input, while `head --1` is a syntax error). And finally, I just checked and the `-#` syntax (where # is a number, of course) is actually *not documented* in the `head` manpage! It's a GNU tool, and they like their non-standard `info` tool, so I had to check the `info head` documentation, where I found the following:

> For compatibility ‘head’ also supports an obsolete option syntax ‘-[NUM][bkm][cqv]’, which is recognized only if it is specified first. NUM is a decimal number optionally followed by a size letter (‘b’, ‘k’, ‘m’) ...

"An obsolete option syntax" doesn't give me confidence that it will continue to be supported going forward. So I've gotten used to using the `-n num` syntax instead.

Reply:
BTW, commit 8bd4169 fixes the comment, which still had a reference to the input parameter we removed earlier.

Reply:
:-D you're too funny!  You had me at the compat reasoning which I can appreciate but a `-1` to get the last line over `tail`?!  That's sheer madness 🤪